### ISSUE TITLE
wasm-check fails on ink! ERC20 example

### ISSUE THREAD

**Issue by peblair** [paritytech/wasm-utils#168]
To reproduce, create a project with the [ink! ERC20](https://substrate.dev/substrate-contracts-workshop/#/2/testing) sample code and then run the following:
```bash
$ cargo install pwasm-utils --features cli
$ cargo contract build
$ wasm-check target/ink/erc20.wasm
All imports should be from env
```
Looking through the substrate codebase, I believe that it is the `wasm-check` tool which is incorrect, as the `sealN` modules (which are what are being used for the imports) should be correct.

**Comment by athei** [comment]
You are correct in your analyzes. Those CLI tools are somewhat lagging behind in features because all the internal users are using the crate directly. I was meaning to give them some love soon.
